ide-plugins.md

```diff
@@ -0,0 +1,34 @@
+# IDE plugins configuration required
+
+## Prettier (automatic formatting code)
+
+If you're using VS Code, install the **_Prettier - Code formatter_** plugin ([for VS Code](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode")).
+
+Next, create a **.prettierrc** file to set basic settings. Our basic settings:
+
+```json
+{
+  "semi": true,
+  "singleQuote": true,
+  "jsxSingleQuote": false,
+  "bracketSpacing": true,
+  "trailingComma": "all",
+  "printWidth": 100,
+  "tabWidth": 2
+}
+```
```

Comment:
We already provide a config for the project in `.prettierrc`. No need for duplication